### PR TITLE
Refs #36330 - Place assets into plugin dir

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -350,8 +350,6 @@ class Application < Rails::Application
       ::Foreman::Plugin.all.select { |p| p.gettext_domain }.each do |plugin|
         domain = plugin.gettext_domain
         Foreman::Gettext::Support.add_text_domain domain, plugin.locale_path
```
Comment (ekohl) [inline config/application.rb]: You could consider making this even shorter
```suggestion
        Foreman::Gettext::Support.add_text_domain plugin.gettext_domain, plugin.locale_path
```

**Review by ekohl — DISMISSED** [review]
Yay simplification.

**Review by ekohl — APPROVED** [review]

File: app/helpers/reactjs_helper.rb [hunk 2]
```diff
@@ -70,7 +70,7 @@ def locale_js_tags
     ::Foreman::Plugin.all.filter_map do |plugin|
       domain = plugin.gettext_domain
       if domain && (FastGettext.translation_repositories[domain]&.available_locales || []).include?(locale)
-        javascript_include_tag("locale/#{locale}/#{domain}")
```
Comment (ofedoren) [inline app/helpers/reactjs_helper.rb:73]: I've just noticed that if I have preferred language in the browser, e.g. en_GB, but a plugin doesn't have js file for that, I won't be able to even log in to change the application language :/

ScreenShot-1688133568083

Comment (ekohl) [reply app/helpers/reactjs_helper.rb:73]: Something I did suspect when reviewing it:
https://github.com/theforeman/foreman/pull/9691#discussion_r1190043816

I think that got lost somewhere in the various iterations the patch went through.

**Pull request merged**